### ISSUE TITLE
[BUG] DAQ_Move with only one axis cannot be defined as slave in preset

### ISSUE THREAD

**Issue by rgeneaux** [PyMoDAQ/PyMoDAQ#485]
### Describe the bug

When a daq_move has only one axis, and is not multiaxes:
```
    is_multiaxes = False
    axes_names = ['Wavelength'] 
```
the `comon_parameters_fun` from move_utility_classes.py automatically hides the multiaxes Parameter group: 
`'title': 'MultiAxes:', 'name': 'multiaxes', 'type': 'group', 'visible': is_multiaxes, ...`

This means that the user cannot set the daq_move as a Slave in a preset, even though there are cases where this is needed - for instance the Spectrometer example from the teaching plugin!

### Reference(s)

```markdown
pymodaq.control_modules.move_utility_classes.py
```

### Environment

```markdown
- OS:Windows
- Pymodaq:4.4.7
- python env:3.11
```

**Comment by seb5g** [comment]
True but you could set the detector as slave...I guess this would only happen with fake/virtual instruments so should we hide it or not?

**Comment by rgeneaux** [comment]
It feels more safe to have it visible always
actually maybe it would be better placed next to the Controller ID

also I wonder, is there a need for the "is multiaxes" tickbox? This seems redundant with the is_mutliaxes defined in the plugin

**Comment by seb5g** [comment]
you're completely right. This all date from old versions that should be refurbished! I'm ont it

**Comment by malik-irain** [comment]
Work in progress in branch [cleaning/controller_id](https://github.com/PyMoDAQ/PyMoDAQ/tree/cleaning/controller_id)

**Comment by seb5g** [comment]
This has been merged in #536 : note that you'll have to recreate anew your presets!
